Assets/Scripts/UI/MinMaxWidth.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    [ExecuteInEditMode]
```

Comment:
Moet deze echt ExecuteInEditMode zijn? Want dat kan serialized changes betekenen en onverwachte changes in de Scene files. Als di bedoelt is voor OnValidate, dat zou niet moeten hoeven omdat OnValidate juist alleen in Edit mode gebeurd